relay-server/src/service.rs

```diff
@@ -435,6 +454,47 @@ pub fn create_redis_pools(configs: RedisPoolConfigs) -> Result<RedisPools, Redis
     }
 }
 
+#[cfg(feature = "processing")]
+fn initialize_redis_scripts_for_pools(redis_pools: &RedisPools) -> Result<(), RedisError> {
+    let project_configs = redis_pools.project_configs.client()?;
+    let cardinality = redis_pools.cardinality.client()?;
+    let quotas = redis_pools.quotas.client()?;
+    let misc = redis_pools.misc.client()?;
+
+    let scripts = [
+        RedisScripts::load_cardinality(),
+        RedisScripts::load_global_quota(),
+        RedisScripts::load_is_rate_limited(),
+    ];
+
+    let pools = [project_configs, cardinality, quotas, misc];
+    for pool in pools {
+        initialize_redis_scripts(pool, &scripts)?;
+    }
+
+    Ok(())
+}
+
+#[cfg(feature = "processing")]
+fn initialize_redis_scripts(
+    mut pooled_client: PooledClient,
+    scripts: &[&RedisScript; 3],
+) -> Result<(), RedisError> {
+    let mut connection = pooled_client.connection()?;
+
+    for script in scripts {
+        // We load on all instances without checking if the script is already in cache because of a
+        // limitation in the connection implementation.
+        redis::cmd("SCRIPT")
```

Comment:
You can do this via [`ScriptInvocation::load`](https://docs.rs/redis/0.27.2/redis/struct.ScriptInvocation.html#method.load)

Reply:
Oh nice, this is great!